src/main/java/org/kohsuke/github/GHCheckRun.java

```diff
@@ -106,14 +109,17 @@ public String getHeadSha() {
      * Gets the pull requests participated in this check run.
      *
      * @return Pull requests of this check run
+     * @throws IOException
+     *             the io exception
      */
-    GHPullRequest[] getPullRequests() throws IOException {
+    public Iterator<GHPullRequest> getPullRequests() throws IOException {
```

Comment:
Sorry, I mispoke.  We want to return an `Iterable`.

```suggestion
    public Iterable<GHPullRequest> getPullRequests() throws IOException {
```

Reply:
Ah right, I guess instead of having the method signature be `Iterable` we can set it to `List` specifically? Updated as such.